### PR TITLE
Add FAQ entry for non-ASCII paths on windows

### PR DESCRIPTION
**Pull request by blowekamp**
Relates to #795 #1255

### CODE REVIEW

**Comment by blowekamp** [comment]
@zivy Please take a look.

**Review by dave3d — APPROVED** [review]

File: docs/source/faq.rst [hunk 1]
```diff
@@ -319,6 +319,37 @@ frequently seem to work with the Enthought Canopy Python
 distribution. However, we recommend compiling SimpleITK
 explicitly against this Python distribution to ensure compatibility.
 
+How do I read an image with a special character in the filename?
+~~~~~~~~~~~~~~~~
+
+Special characters are non-ANSI characters, which require a non-ASCII encoding to for representation. Unicode is the
```
Comment (zivy) [inline docs/source/faq.rst:325]: "...encoding to for representation." -> "...encoding for representation."

File: docs/source/faq.rst [hunk 2]
```diff
@@ -319,6 +319,37 @@ frequently seem to work with the Enthought Canopy Python
...
+modern standard with UTF-8 being the most common. The underlying ITK API call use bytes to represent the filename which
```
Comment (zivy) [inline docs/source/faq.rst:326]: call->calls: "The underlying ITK API call use bytes" -> "The underlying ITK API calls use bytes" 
or
use->uses: "The underlying ITK API call use bytes" -> "The underlying ITK API call uses bytes"

File: docs/source/faq.rst [hunk 3]
```diff
@@ -319,6 +319,37 @@ frequently seem to work with the Enthought Canopy Python
...
+is compatible with UTF-8. This works on modern POSIX and MacOS systems. However, on Windows, the default encoding is
+an alternative 8-bit encoding and wide UTF-16 is used to represent Unicode code points.
+
+The Windows Operating Systems since about 2020 can be configured to use UTF-8 as the default encoding. This setting
+can be found under "All Settings -> Time & Language -> Language -> "Administrative Language Settings" and the option
+"Beta: Use Unicode UTF-8 for worldwide language support". However, this may change in the future and may have unintended
+consequences for other languages.
+
+An alternative, is to set the character locale to UTF-8 in the Python script. This can be done by with the following
```
Comment (zivy) [inline docs/source/faq.rst:335]: "This can be done by with the following..." -> "This can be done with the following..."

**Review by zivy — APPROVED** [review]
Some text changes suggested. Nice workaround for Windows Python users.

**Pull request merged**